packages/config/config/devices/0x010f/fgs214.json

```diff
@@ -35,121 +35,297 @@
 	"paramInformation": [
 		{
 			"#": "1",
-			"label": "Remember relays state",
-			"description": "This parameter determines the state of relays power supply failure (e.g. power outage). auto OFF and flashing modes the parameter not relevant and the relay will always remain off.",
+			"label": "Relays State After Power Failure",
 			"valueSize": 1,
 			"minValue": 0,
 			"maxValue": 2,
 			"defaultValue": 1,
-			"unsigned": true,
 			"allowManualEntry": false,
 			"options": [
 				{
-					"label": "Relays remain switched off after restoring power",
+					"label": "Always off",
 					"value": 0
 				},
 				{
-					"label": "Restore remembered state of relays after restoring power",
+					"label": "Previous state",
 					"value": 1
 				},
 				{
-					"label": "Restore remembered state of relays after restoring power, but for toggle switches (parameter 20/21 set to 1) set the same state as the current state of the switches",
+					"label": "Same as switches",
 					"value": 2
 				}
 			]
 		},
 		{
 			"#": "20",
-			"label": "S1 input – switch type",
-			"description": "S1 - Inputs type configuration",
+			"label": "S1 Input – Switch Type",
 			"valueSize": 1,
 			"minValue": 0,
 			"maxValue": 2,
-			"defaultValue": 2,
-			"unsigned": true,
+			"defaultValue": 0,
 			"allowManualEntry": false,
 			"options": [
 				{
 					"label": "Momentary switch",
 					"value": 0
 				},
 				{
-					"label": "Toggle switch (contact closed - On, contact opened - OFF)",
+					"label": "Toggle switch synchronized",
 					"value": 1
 				},
 				{
-					"label": "Toggle switch (device changes status when switch changes status)",
+					"label": "Toggle switch with memory",
```

Comment:
I'm not 100% happy with these, the original is a bit verbose, but the shortened version is not self-explanatory. What about something along these lines?

value 1 -> `"Toggle switch (synchronize output)"`
value 2 -> `"Toggle switch (toggle output)"`

Reply:
Indeed, that's way better.